### PR TITLE
Add theme "separation"

### PR DESCRIPTION
**Pull request by roflcopter4**
Adds a theme which displays a horizontal bar across the terminal for each prompt, visually separating the output of each command. Displays return status (if not 0), username, hostname, and the working directory on the second line. If this string is longer than two fifths of the screen, the prompt is split into a third line, ensuring plenty of space to type commands.

### CODE REVIEW

File: packages/separation [hunk 1]
```diff
@@ -0,0 +1,5 @@
+type        = theme
+repository  = https://github.com/roflcopter4/separation
+maintainer  = RoflCopter4 <brendan.leason.4@gmail.com>
+description = An oh-my-fish theme for those who like the output of each 
+              command to be clearly distinguishable from the rest.
```
Comment (sagebind) [inline packages/separation]: I don't think multi-line descriptions will work in the manifest parser. You might want to put the description all on one line here.

**Comment by sagebind** [comment]
Thanks for submission, that is a really cool idea for a theme! Other than the manifest issue, looks good to me.

**Comment by roflcopter4** [comment]
Thanks for the feedback. I fixed up the description field to be one line. I hope the length of that one line won't be an issue (I noticed that a few other packages also have similarly long descriptions).

I also have no idea whether a prompt like this would be popular. I've never seen one quite like it in any of the theme lists for a shell. Some do feature a horizontal bar, but they always have some kind of text somewhere on that line as well. If this is accepted, I'd be quite interested to see if anyone actually uses it.

**Comment by oranja** [comment]
OMF doesn't collect any statistics about its usage, but I personally like this theme. Some kind of eye catching separation between commands is something I look for in any theme I use.

You can also watch the Stars count on your repository and it will give you an indication of how many people were willing to go out of their way to show their appreciation for your work.

**Pull request merged**